.github/workflows/pytest.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Pytest, McCabe, and Coverage
         run: |
           cd my_project
-          pipenv install --dev --deploy --skip-lock
+          pipenv install --dev --deploy
```

Comment:
why did we make this change?

Reply:
yeaaa....pip install couldn't possibly have the --dev flag. Lets merge this as-is and open a github issue to figure out a way to bring this back using pipenv